numba/unicode.py

```diff
@@ -531,6 +531,20 @@ def find_impl(a, b):
         return find_impl
 
 
+@overload_method(types.UnicodeType, 'rfind')
+def unicode_rfind(s, substr):
```

Comment:
Missed optional start and end args
Also need to check type of these args, like is done for count: https://github.com/numba/numba/blob/master/numba/unicode.py#L903-L909

Reply:
I will do it.